### ISSUE TITLE
dropdown to switch race for cumulative itemized campaign contributions graph

### ISSUE THREAD

**Issue by ianaroot** [openoakland/opendisclosure#378]
add dropdown of various races with event binding to rerender the graph based on contribution data for that particular race.

**Comment by ianaroot** [comment]
@macfarlandian, lauren told me to tag you on this issue and maybe we could collaborate

**Comment by macfarlandian** [comment]
yeah – i don't really know what the current implementation is, but my thought was to use the C3 chart library, which gives you most of the interactivity lauren was talking about for free: http://c3js.org/samples/timeseries.html

we'd just need to add a dropdown to load some new data

**Comment by evanwolf** [comment]
question about payload. more than half of our users will be accessing this on mobile. so...

- Bandwidth: Is it cheaper (in bytes) to download data and render in the browser or to pre-render on the server and download images? we can achieve similar animation through css+js as through C3. 
- Connectivity: Can we cache data in the browser for offline viewing?

**Comment by macfarlandian** [comment]
i'm also wondering where this particular feature fits into the grand scheme of redesigning the homepage ... if individual races are going to have their own pages that look kind of like what the mayor-only homepage currently looks like, a dropdown to switch races might not be super relevant anymore.

**Comment by lla2105** [comment]
Hi @macfarlandian good point on the dropdown not being as relevant now with the homepage redesign. @elinaru what do you think? since you have more of an idea of what our redesign homepage is going to look like.
